bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,23 @@ metadata:
   annotations:
     alm-examples: |-
       [
+        {
+          "apiVersion": "nac.oadp.openshift.io/v1alpha1",
+          "kind": "NonAdminBackup",
+          "metadata": {
+            "labels": {
+              "app.kubernetes.io/created-by": "oadp-operator",
+              "app.kubernetes.io/instance": "nonadminbackup-sample",
+              "app.kubernetes.io/managed-by": "kustomize",
+              "app.kubernetes.io/name": "nonadminbackup",
+              "app.kubernetes.io/part-of": "oadp-operator"
+            },
+            "name": "nonadminbackup-sample"
+          },
+          "spec": {
+            "backupSpec": {}
```

Comment:
I know this is copied from the non-admin controller, but it should contain some valid example? Of course we can have it in iteration.

Reply:
this need to be done in NAC repo, but can be future improvement

Reply:
what we have for backup https://github.com/openshift/oadp-operator/blob/master/config/samples/backup.yaml

Reply:
will be worked in the future in another PR